### DISCUSSION TITLE
Which is the only continent in the world without an active volcano?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#779] [Galaxy Trivia] [answered]
**Bonus:** Which continent has the greatest concentration of volcanoes?

**Comment by SANKARAMDAS** [comment]
Australia is the only continent without any current volcanic activity, but it hosts one of the world's largest extinct volcanoes, the Tweed Volcano. Rock dating methods indicate that eruptions here lasted about three million years, ending about 20 million years ago.

Antarctica has the greatest concentration of volcanoes in the world, according to a new study.

**Comment by peterbaccaro** [comment]
The continent without an active volcano is **Australia**. While Australia has a rich geological history and contains many ancient volcanic features, it does not currently have any active volcanoes. The most recent volcanic activity on the Australian continent occurred tens of thousands of years ago.

The continent with the greatest concentration of volcanoes is **Asia**. Asia is home to the Pacific Ring of Fire, which is an area where a large number of earthquakes and volcanic eruptions occur. It stretches along the coasts of several countries in the Asia-Pacific region, including Japan, Indonesia, the Philippines, and Kamchatka (Russia). This region is known for its high volcanic activity and has a significant number of active and dormant volcanoes. Some notable volcanoes in Asia include Mount Fuji in Japan, Mount Merapi in Indonesia, and Mount Pinatubo in the Philippines.

**Comment by ParanoidUser** [comment]
Any trustworthy source we can count on?

**Reply by SANKARAMDAS** [reply]
NASA (.gov)
https://photojournal.jpl.nasa.gov/catalog/PIA06664#:~:text=Australia%20is%20the%20only%20continent,about%2020%20million%20years%20ago.

**Reply by ParanoidUser** [reply]
> NASA (.gov) https://photojournal.jpl.nasa.gov/catalog/PIA06664#:~:text=Australia%20is%20the%20only%20continent,about%2020%20million%20years%20ago.

  I have no doubts about 🇦🇺, but on 🇦🇶 🐧🐧

**Reply by SANKARAMDAS** [reply, marked as answer]
AQ 🐧🐧

https://www.theguardian.com/world/2017/aug/12/scientists-discover-91-volcanos-antarctica

https://qz.com/1052938/antarctica-has-the-greatest-concentration-of-volcanoes-in-the-world-according-to-a-new-study
